connect.php

```diff
@@ -1,10 +1,9 @@
 <?php
```

Comment:
Take account of the following points
1. do not remove include statement.
2. DBPASSWORD shouldn't be within single quotes ''